.github/CODEOWNERS

```diff
@@ -33,3 +33,10 @@ specs @rajsite @jattasNI @atmgrifter00 @mollykreis
 
 # Change files don't need explicit reviewers
 /change
```

Comment:
nit: move your new section above `/change` since we want the change rule to have the highest priority. I don't think it'll make a practical difference but keeps that intent clear.